R/helpers_data.R

```diff
@@ -0,0 +1,235 @@
+#' Helper Function to Blind Plot Data
+#'
+#' @param df (`GeneralData`)\cr The data to be blinded
+#' @param blind (`flag`)\cr Should the data be blinded?
+#' @param has_placebo (`flag`)\cr Does the data contain a placebo dose?
+#' @param pbo_dose (`positive_number`)\cr The dose to be taken as placebo.
+#' Ignored if `has_placebo` is `FALSE`
+#' @returns The blinded data
```

Comment:
```suggestion
#' @param df (`data.frame`)\cr the data to be blinded.
#' @param blind (`flag`)\cr whether the data should be blinded.
#' @param has_placebo (`flag`)\cr whehter the data contains a placebo dose.
#' @param pbo_dose (`positive_number`)\cr the dose to be taken as placebo; Ignored if `has_placebo` is `FALSE`.
#' @returns The blinded data
```
because `df` is  a data.frame I think based on how it is used inside?